src/wcstringutil.h

```diff
@@ -133,21 +133,26 @@ wcstring_list_t split_string_tok(const wcstring &val, const wcstring &seps,
 /// Join a list of strings by a separator character.
 wcstring join_strings(const wcstring_list_t &vals, wchar_t sep);
 
-inline wcstring to_string(long x) {
+// prevents 'ambiguous' compiler error in OpenBSD clang
+#ifndef __OpenBSD__
+inline wcstring to_wcstring(long x) {
     wchar_t buff[64];
     format_long_safe(buff, x);
     return wcstring(buff);
 }
+#endif
 
-inline wcstring to_string(unsigned long long x) {
+inline wcstring to_wcstring(unsigned long long x) {
     wchar_t buff[64];
     format_ullong_safe(buff, x);
     return wcstring(buff);
 }
 
-inline wcstring to_string(int x) { return to_string(static_cast<long>(x)); }
-
-inline wcstring to_string(size_t x) { return to_string(static_cast<unsigned long long>(x)); }
+// prevents 'ambiguous' compiler error in OpenBSD clang
+#ifndef __OpenBSD__
```

Comment:
No need to have two `#ifndef __OpenBSD__` blocks, just put these above into the other one.